### PR TITLE
Rename repository to ACCESS-docs, Update links

### PR DESCRIPTION
**Pull request by rbeucher**
Rename repository to ACCESS-docs, Update links

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 site_name: ACCESS Hive
```
Comment (aidanheerdegen) [inline mkdocs.yml:1]: I think this needs to change too

Comment (rbeucher) [reply mkdocs.yml:1]: That will change the name on the website I think.

File: docs/call_contribute.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 ???+ warning "We need you to contribute!"
-    The ACCESS Hive is a community resource that is a work in progress. Can you help to add content to this page? We’d love to receive your contribution. See our [contributing guidelines](https://access-hive.github.io/ACCESS-Hive/about/contribute) for details of how to provide content. You can also [open an issue](https://github.com/ACCESS-Hive/ACCESS-Hive/issues) highlighting any content you’d like us to provide but aren’t able to contribute yourself.
\ No newline at end of file
+    The ACCESS Hive is a community resource that is a work in progress. Can you help to add content to this page? We’d love to receive your contribution. See our [contributing guidelines](https://access-hive.github.io/ACCESS-Hive/about/contribute) for details of how to provide content. You can also [open an issue](https://github.com/ACCESS-Hive/ACCESS-docs/issues) highlighting any content you’d like us to provide but aren’t able to contribute yourself.
```
Comment (aidanheerdegen) [inline docs/call_contribute.md:2]:
```suggestion
    ACCESS docs is a community resource that is a work in progress. Can you help to add content to this page? We’d love to receive your contribution. See our [contributing guidelines](https://access-hive.github.io/ACCESS-Hive/about/contribute) for details of how to provide content. You can also [open an issue](https://github.com/ACCESS-Hive/ACCESS-docs/issues) highlighting any content you’d like us to provide but aren’t able to contribute yourself.
```

File: mkdocs.yml (hunk 1 above)
Comment (aidanheerdegen) [reply mkdocs.yml:1]: Yeah don't we want this? I'm confused.

**Comment by rbeucher** [comment]
I'm closing this one and will open another PR

**Pull request closed without merging**